ILRepack/MappingHandler.cs

```diff
@@ -27,13 +27,8 @@ public override int GetHashCode()
                 return scope.GetHashCode() + name.GetHashCode();
             }
 
-            public override bool Equals(object obj)
```

Comment:
They also say we should override == and != :smile: 
This isn't a general-purpose type (it should be private), hence I'm not sure it's necessary to add those, they'll never be used.